extensions.json

```diff
@@ -287,6 +287,17 @@
             "method": "git",
             "directory": "."
         },
+        {
+            "name": "RSS-Bridge",
+            "author": "Devon Hess",
+            "description": "Run URLs through <a href=\"https://github.com/rss-bridge/rss-bridge\">RSS-Bridge</a> detection.",
```

Comment:
I believe it is the first example of HTML use in the description field, which we support, but with which we should be extra careful.